src/interpreter_funcs/join_meshes.rs

```diff
@@ -41,11 +49,19 @@ impl Func for FuncJoinMeshes {
     fn call(
         &mut self,
         args: &[Value],
-        _log: &mut dyn FnMut(LogMessage),
+        log: &mut dyn FnMut(LogMessage),
     ) -> Result<Value, FuncError> {
-        let meshes = args.iter().map(|a| a.unwrap_mesh());
+        let meshes = vec![args[0].unwrap_mesh(), args[1].unwrap_mesh()];
```

Comment:
Let's not allocate a vector. You can use an array or a slice instead.

Reply:
help please?

```rust
let meshes = &[args[0].unwrap_mesh(), args[1].unwrap_mesh()];
let value = tools::join_multiple_meshes(meshes);
```

```
error[E0271]: type mismatch resolving `<&[&mesh::Mesh; 2] as std::iter::IntoIterator>::Item == &mesh::Mesh`
   --> src\interpreter_funcs\join_meshes.rs:58:21
    |
58  |         let value = tools::join_multiple_meshes(meshes);     
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected &mesh::Mesh, found struct `mesh::Mesh`
    | 
   ::: src\mesh\tools.rs:333:8
    |
333 | pub fn join_multiple_meshes<'a, I>(meshes: I) -> Mesh        
    |        --------------------
334 | where
335 |     I: IntoIterator<Item = &'a Mesh>,
    |                     --------------- required by this bound in `mesh::tools::join_multiple_meshes`
    |
    = note: expected type `&&mesh::Mesh`
               found type `&mesh::Mesh`
```